### ISSUE TITLE
What data is the minimum data needed to track an individual attendance?

### ISSUE THREAD

**Issue by corlaez** [corlaez/asistencia#1]
The less data needed the faster and more flexible the tracking can be.

Do we simply need to count heads? Do we need to know sex or age group? Do we need to know names? ids?

The more data is needed we may need to register users.

**Comment by JessicaMP** [comment]
Regarding follow-up data, personal data of the assistant (Full name, age, address, telephone number, family, etc.), academic data, Bible studies, etc.

I think the most appropriate thing is to handle it by ids.

If it is very possible to add more users, suspend them in case they decide to change churches or no longer attend.

**Comment by corlaez** [comment]
What do you mean with family? Is that really needed?

**Comment by JessicaMP** [comment]
They believe necessary.

**Comment by corlaez** [comment]
Can you expand what family is?

**Comment by JessicaMP** [comment]
Yes, information about parents, brothers, sisters maybe attends the church another member of your family for example grandmother, aunt .We will cant categorize in extra information.

**Comment by corlaez** [comment]
And why do we want to track family relationships? 
They may be tricky to track. I would like to understand why they want this.
We could add it but it is hard to maintain and it doesn't seem needed to track attendance.
I am tempted to say this is something that is outside of our scope.
The fact that they believe it is necessary shouldn't really be enough for us adding the feature as we are aiming for a generic solution.

**Comment by JessicaMP** [comment]
Ok, but maybe you could be considered in input or textarea observations

**Comment by corlaez** [comment]
Okay. Generic text area with comments makes sense.
